BioForm/profiles/views.py

```diff
@@ -64,4 +55,52 @@ def login_user(request):
 
 def logout_user(request):
     logout(request)
-    return redirect("login")
\ No newline at end of file
+    return redirect("login")
+
+def create_record(request):
```

Comment:
Currently, not a single endpoint is secure, you can create a new record by directly going to `/profiles/create/`. Make sure a user can't access these pages without authentication.